platforms/hyperledger-fabric/charts/invoke_chaincode/templates/invoke_chaincode.yaml

```diff
@@ -111,6 +117,37 @@ spec:
           echo "${CACERTS}" >> ${OUTPUT_PATH}/cacerts/ca.crt
           echo "${KEYSTORE}" >> ${OUTPUT_PATH}/keystore/server.key
           echo "${SIGNCERTS}" >> ${OUTPUT_PATH}/signcerts/server.crt
+          
+          ######################################################################
+
+          version2=`echo $NETWORK_VERSION | grep -c 2.`
+          if [ $version2 = 1 ];then
+            #Downloading Certificates for all admin users
+            echo $ENDORSING_ORG_NAMES
+            for ns in $ENDORSING_ORG_NAMES
+            do
+              SECRETPREFIX=secret/crypto/peerOrganizations/${CREATOR_NS}/endorsers/${ns}/users/admin
+              echo "Fetching certificates for org: $ns"
+              echo $SECRETPREFIX
```

Comment:
Maybe add some more descriptive logs here:
echo "Secret prefix for org: $SECRETPREFIX
Rest for the other echo's of just variables, will make the logs more readable & clear